pkg/util/managedcluster/managedcluster.go

```diff
@@ -44,30 +43,10 @@ func getRestConfigFromV1Config(kc *v1.Config) (*rest.Config, error) {
 	return kubeconfig.ClientConfig()
 }
 
-// ClientsetFromV1Config takes a v1 config and returns a Clientset
-func ClientsetFromV1Config(config *v1.Config) (*kubernetes.Clientset, error) {
-	restconfig, err := getRestConfigFromV1Config(config)
-	if err != nil {
-		return nil, err
-	}
-
-	return kubernetes.NewForConfig(restconfig)
-}
-
 // WaitForHealthz takes a context, v1 config.
 // It waits for the cluster to respond to healthz requests.
-func WaitForHealthz(ctx context.Context, config *v1.Config, log *logrus.Entry) error {
-	restconfig, err := getRestConfigFromV1Config(config)
-	if err != nil {
-		return err
-	}
-
-	t, err := rest.TransportFor(restconfig)
-	if err != nil {
-		return err
-	}
-
+func WaitForHealthz(ctx context.Context, cs *api.OpenShiftManagedCluster, log *logrus.Entry) error {
```

Comment:
@jim-minter,  I know we previously discussed passing in a `cs *api.OpenShiftManagedCluster` object.  Do you think we could ultimately get away with passing just the `cs.Properties.FQDN+"/healthz"` since that is what is created immediately upon receiving the `cs`.  Thoughts?